config/src/main/java/com/alibaba/nacos/config/server/service/PersistService.java

```diff
@@ -508,7 +509,8 @@ public Boolean doInTransaction(TransactionStatus status) {
                     String configTags = configAdvanceInfo == null ? null : (String) configAdvanceInfo.get("config_tags");
                     addConfiTagsRelationAtomic(configId, configTags, configInfo.getDataId(), configInfo.getGroup(),
                         configInfo.getTenant());
-                    insertConfigHistoryAtomic(0, configInfo, srcIp, srcUser, time, "I");
+                    String type = configAdvanceInfo == null ? null : (String) configAdvanceInfo.get("type");
```

Comment:
I think if type not set, The text can be set to the default value of type

Reply:
Good advice. I'll change it.